jena-fuseki2/jena-fuseki-webapp/src/test/java/org/apache/jena/fuseki/TestAdmin.java

```diff
@@ -278,7 +278,29 @@
         deleteDataset(dsTest);
     }
 
-    // Sync task testing
+    @Test public void sleep_1() {
+        String x = execSleepTask(null, 1);
+    }
+
+    @Test public void sleep_2() {
+        try {
+            String x = execSleepTask(null, -1);
+            fail("Sleep call unexpectedly succeed");
+        } catch (HttpException ex) {
+            assertEquals(400, ex.getStatusCode());
+        }
+    }
+
+    @Test public void sleep_3() {
+        try {
+            String x = execSleepTask(null, 20*1000+1);
```

Comment:
Would be nicer if this just referred to the constant in `ActionSleep` so if we ever change that the test doesn't break